### PR TITLE
Migrate apt_repo to community.general

### PR DESCRIPTION
**Pull request by sivel**
##### SUMMARY
Migrate apt_repo to community.general. Fixes https://github.com/ansible/ansible/issues/68637

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
plugins/modules/packaging/os/apt_repo.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/modules/packaging/os/apt_repo.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+---
+module: apt_repo
+short_description: Manage APT repositories via apt-repo
+description:
+  - Manages APT repositories using apt-repo tool.
+  - See U(https://www.altlinux.org/Apt-repo) for details about apt-repo
+notes:
+    - This module works on ALT based distros.
+    - Does NOT support checkmode, due to a limitation in apt-repo tool.
+version_added: "2.8"
```
Comment (felixfontein) [inline plugins/modules/packaging/os/apt_repo.py:25]:
```suggestion
```

Comment (mkrizek) [inline plugins/modules/packaging/os/apt_repo.py:25]: FWIW `migrate.py` removed all `version_added` from `DOCUMENTATION` during the migration so this should be removed too.

Comment (sivel) [reply plugins/modules/packaging/os/apt_repo.py:25]: Oh, that's interesting

Comment (sivel) [reply plugins/modules/packaging/os/apt_repo.py]: done

Comment (felixfontein) [reply plugins/modules/packaging/os/apt_repo.py]: It didn't remove them for return values, though..

Comment (mkrizek) [reply plugins/modules/packaging/os/apt_repo.py]: Correct, `EXAMPLES` and `RETURN` were left intact. Unfortunately there was only so much we were able to automate in time.

**Comment by felixfontein** [comment]
I guess there were no tests?

**Comment by sivel** [comment]
> I guess there were no tests?

There were not.

**Pull request merged**